### ISSUE TITLE
[I] Strange highlighting of the "All Notes" folder

### ISSUE THREAD

**Issue by Katarn** [pbek/QOwnNotes#2488]
I just made a screencast.

https://user-images.githubusercontent.com/6339978/156758645-933b054f-a6ad-4c29-ab4a-f8e3604b277c.mp4

#### Information about the application, settings and environment

<details><summary>Expand</summary>

QOwnNotes Debug Information
===========================

## General Info

**Current Date**: `Пт мар 4 13:35:21 2022`
**Version**: `22.3.0`
**Build date**: `Mar  3 2022`
**Build number**: `955`
**Platform**: `windows`
**Operating System**: `Windows 8.1 Version 6.3 (Build 9600)`
**Build architecture**: `x86_64`
**Current architecture**: `x86_64`
**Release**: `GitHub Actions`
**Qt Version (build)**: `5.15.2`
**Qt Version (runtime)**: `5.15.2`
**Portable mode**: `no`
**Qt Debug**: `0`
**Locale (system)**: `ru_RU`
**Locale (interface)**: *empty*
**Primary screen resolution**: `1920x1080`
**Screen resolution(s)**: `1920x1080, 1920x1080`
**Icon theme**: `breeze-dark-qownnotes`
**Notes in current note folder**: `252`
**Calendar items**: `0`
**Enabled scripts**: `2`
</details>

**Comment by pbek** [comment]
Yes, you cannot select that item. That's on purpose. It's not a real sub-folder.

**Comment by Katarn** [comment]
Hm, I got it. What about two columns, each with a separate selection? More like a table.

**Comment by pbek** [comment]
A QTreeWidget can't be used as a table. 😀  And I'm not sure what a table would solve in this case or who actually would expect a table or would want to spend the space for it 😉

**Comment by Katarn** [comment]
No, I'm talking about something else. :) The video shows that two columns - left and right - have a separate selection highlight.

**Comment by pbek** [comment]
Ah, you didn't described that effect in your issue. It is very hard to see in the video. 😉  I didn't find a way around it. It's how Qt handles QTreeWidgetItems that can't be selected.

**Comment by Katarn** [comment]
I was thinking, maybe from the point of view of the UX, the disabled items in the list are wrong? Offhand, I can't remember that somewhere this was in the interfaces of operating systems, the same Windows or, for example, KDE.

**Comment by pbek** [comment]
Which disabled items? "All notes" isn't disabled. Just the selection is turned off for that item.

**Comment by Katarn** [comment]
I have not seen such behavior in the lists anywhere else. Slightly unintuitive behavior, IMHO. :) I click on the "All notes" item in the list and all selected items are deselected. A new selection does not appear in the list.

**Comment by Katarn** [comment]
![image](https://user-images.githubusercontent.com/6339978/156899309-6cee34be-08fc-4a0d-a7a6-fbcf18d90ce4.png)
![image](https://user-images.githubusercontent.com/6339978/156899319-afb03b90-ee4b-439d-abcf-7b9e8f9c7563.png)

Here's what else I noticed. This is not an "All Notes" list item. Maybe there is an answer to the question of how to remove this rectangle:

https://www.qtcentre.org/threads/31694-Get-rid-of-the-stupid-frustrating-focus-rectangle-in-QTreeWidget

**Comment by pbek** [comment]
Thank you for researching. Maybe someone wants to experiment with that at some time.

**Comment by pbek** [comment]
I can't reproduce that anymore as of 26.2.13 on Qt 6.10.1.
